libtock/humidity.c

```diff
@@ -9,28 +9,40 @@ struct data {
 static struct data result = { .fired = false };
 
 // Internal callback for faking synchronous reads
-static void cb(int humidity,
-               __attribute__ ((unused)) int unused,
-               __attribute__ ((unused)) int unused1,
-               void* ud) {
+static void upcall(int humidity,
```

Comment:
Should the comment above this also change to `upcall`?

Reply:
The plan is to change everything to upcall, to distinguish it from general callbacks within the kernel. Otherwise it starts to be confusing, whether you're talking about a `Callback` or a callback.

A callback is just a function that's called back on the caller; an upcall is when the kernel invokes a function on a userspace process.